src/Shared/runtime/Http2/Hpack/DynamicTable.cs

```diff
@@ -1,5 +1,5 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
```

Comment:
The `Shared/runtime` sources are mirrored from `dotnet/runtime` and are correctly MIT licensed.